tests/cases/conformance/types/mapped/mappedTypeRelationships.ts

```diff
@@ -192,3 +192,21 @@ function f90<T extends { x: number }>() {
 function f<T extends { x: {} }>(): Partial<T> {
     return undefined! as T;
 }
+
+// #32365
+interface SettingsTypes {
+  audio: {
+    volume: string;
+  };
+  video: {
+    resulution: string;

```

Comment:
I know it was part of the original issue - but can you fix this to `resolution`?